src/openvic-simulation/politics/Issue.cpp

```diff
@@ -178,19 +179,22 @@ bool IssueManager::_load_issue(
 	ModifierManager const& modifier_manager, RuleManager const& rule_manager, std::string_view identifier,
 	IssueGroup const* group, ast::NodeCPtr node
 ) {
-	using enum Modifier::modifier_type_t;
-
+	fixed_point_t war_exhaustion_effect = 0;
 	ModifierValue values;
 	RuleSet rules;
 	bool jingoism = false;
 
-	bool ret = modifier_manager.expect_modifier_value_and_keys(
-		move_variable_callback(values),
-		ISSUE,
+	bool ret = NodeTools::expect_dictionary_keys_and_default(
+		modifier_manager.expect_base_country_modifier(values),
 		"is_jingoism", ZERO_OR_ONE, expect_bool(assign_variable_callback(jingoism)),
-		"rules", ZERO_OR_ONE, rule_manager.expect_rule_set(move_variable_callback(rules))
+		"rules", ZERO_OR_ONE, rule_manager.expect_rule_set(move_variable_callback(rules)),
+		"war_exhaustion_effect", ZERO_OR_ONE, expect_fixed_point(assign_variable_callback(war_exhaustion_effect))
 	)(node);
 
+	if(ret && war_exhaustion_effect != fixed_point_t::_0()) {
```

Comment:
Rather than checking it isn't zero, it would be better to get rid of the variable and instead use a custom callback that prints the warning message whenever `war_exhaustion_effect` is found regardless of value.

Reply:
That is a better solution. Thanks for the suggestion.